backend/src/services/tests/muralcollection.test.ts

```diff
@@ -0,0 +1,144 @@
+import { database } from "../../config/database";
+import { MuralCollection } from "../../models/muralcollection.model";
+import { MuralCollectionService } from "../muralcollection.service";
+import { Mural } from "../../models/mural.model";
+import { Borough } from "../../models/borough.model";
+import { Artist } from "../../models/artist.model";
+
+beforeAll(async () => {
+  await Mural.belongsTo(Borough, {
```

Comment:
I know in the other tests we use 'await' here, but it is not actually necessary, it does nothing for this type of statement, so might as well remove.